packages/expo/src/winter/ImportMetaRegistry.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import { getBundleUrl } from '../utils/getBundleUrl';
+
+/**
+ * Registry to handle import.meta functionality for React Native environment
+ * Similar to how it works in the web, but adapted for the RN context
+ * https://github.com/wintercg/import-meta-registry
+ */
+class ImportMetaRegistryClass {
+  public readonly url = getBundleUrl();
+
+  public readonly env = process.env;
+}
+
+export const ImportMetaRegistry = new ImportMetaRegistryClass();
```

Comment:
Feels a little more straightforward to me to do:
```suggestion
export const ImportMetaRegistry = {
  get url() {
    return getBundleUrl();
  }
  
  get env() {
    return process.env;
  }
};
```
Up to you.